deployment/helm/templates/alert.yaml

```diff
@@ -131,31 +155,26 @@ spec:
   type: ClusterIP
 ---
 
-{{- if and .Values.enablePersistentStorage (not .Values.persistentVolumeClaimName) }}
-apiVersion: v1
-kind: PersistentVolumeClaim
+{{ if and .Values.exposeui (eq .Values.exposedServiceType "Openshift" ) -}}
```

Comment:
can you pls change from Openshift to OpenShift?

Reply:
Done!